tools/enable_mount_volumes.py

```diff
@@ -16,7 +16,6 @@
 import logging
 import os
 import os.path
-#import pprint
 import sys
 import time
 import uuid
```

Comment:
The `fabric` imports below are not part of our standard tooling. As far as I can tell, this is a deprecated API in any case.

This is the reason for the additional `pylint` failure.